src/rgw/rgw_lc.cc

```diff
@@ -1525,7 +1521,9 @@ int RGWLC::bucket_lc_process(string& shard_id, LCWorker* worker,
     return 0;
   }
 
-  int ret = driver->get_bucket(this, nullptr, bucket_tenant, bucket_name, &bucket, null_yield);
+  int ret = driver->load_bucket(this, nullptr,
+                                rgw_bucket_key(bucket_tenant, bucket_name),
```

Comment:
should this be rgw_bucket rather than rgw_bucket_key?

Reply:
good catch, thanks